pulp_ansible/tests/functional/conftest.py

```diff
@@ -99,6 +114,18 @@ def _ansible_repo_factory(**kwargs):
     return _ansible_repo_factory
 
 
+@pytest.fixture
+def sync_repo_factory(ansible_repo_api_client):
```

Comment:
Would it be useful if this factory created the repository and sync it and then return?
In the current state it looks like a sync_helper to me. Also be careful with the name. pytest put's all fixture names in the same scope.

Reply:
Renamed to `ansible_sync_factory`. Also added auto repo creation if not supplied.